staging/src/k8s.io/kube-apiextensions-server/test/integration/testserver/resources.go

```diff
@@ -127,6 +127,24 @@ func CreateNewCustomResourceDefinition(customResourceDefinition *apiextensionsv1
 	if err != nil {
 		return nil, err
 	}
+
+	// the REST storage isn't created until you hit the endpoint (the list does this),
+	// at which point the storage cacher for the CR starts asynchronously.
```

Comment:
would it not be better to inject the delay at the point where we build the rest storage? we'd at least have the possibility of fixing the coordination between the list and the cache server-side instead of making multiple clients add first-use special casing